app/services/standardize_address_service.rb

```diff
@@ -0,0 +1,52 @@
+require 'net/http'
+require 'uri'
+
+# PLEASE do NOT use this service to standardize addresses in bulk (do NOT use in backfills)
+# as that violates the terms of agreement with USPS and will result in access being revoked.
+class StandardizeAddressService
+  def self.standardize_address_on_intake(intake)
+    build_standardized_address(intake.street_address, intake.city, intake.state, intake.zip_code)
```

Comment:
the new mocks had us add a "street_address2" -- a separate line to add apartment number. do we need to pass that in as well?